twitchio/ext/eventsub/server.py

```diff
@@ -51,7 +51,7 @@ async def delete_all_active_subscriptions(self):
         # A convenience method
         active_subscriptions = await self.get_subscriptions("enabled")
         for subscription_id in active_subscriptions:
-            await self.delete_subscription(subscription_id)
+            await self.delete_subscription(subscription_id.id)
```

Comment:
`..._id.id` is anti-pattern, no?

I'd rename `subscription_id` to `subscription`.